tests/cases/fourslash/quickInfoPropertyTag.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+
+// @Filename: /a.js
+/////**
+//// * @typedef I
+//// * @property {number} x Doc
```

Comment:
Can you make the documentation longer and multiline, with varying indents, to make sure that it works correctly?

Reply:
#21123